apps/learner/src/routes/content/[id]/+page.svelte

```diff
@@ -9,6 +9,7 @@
   const { data } = $props();
 
   let returnTo = $state('/');
+  let expanded = $state(false);
```

Comment:
If it is a state, should we name as `isExpanded` instead? WDYT?

Reply:
If I rename this to `isExpanded` then should I rename the function below from `toggleExpanded` to `toggleIsExpanded` or we can leave it as it is?

Reply:
It should rename to `toggleIsExpanded` too for consistency and clarity.

Reply:
Okay! We can try that naming first.